generators/spa/index.js

```diff
@@ -14,7 +14,8 @@ module.exports = class extends DnnGeneratorBase {
         choices: [
           { name: 'Angular', value: 'Angular' },
           { name: 'ReactJS', value: 'ReactJS' },
-          { name: 'VueJS', value: 'VueJS' }
+          { name: 'Vue 2', value: 'VueJS' },
```

Comment:
@alejoroman0605 You ignored my note on this. I asked you to change this to `Vue 2` (if it's Vue 2.xx).  CC: @ismaelTartola

Reply:
@WillStrohl I did not ignore this comment, what I did was change the name, which is what is displayed in the console, but I immediately changed the value as well.

Reply:
@alejoroman0605 When I looked at this before your response, the values did not appear to be updated.